target_postgres/tests/test_standard_target.py

```diff
@@ -326,6 +326,42 @@ def test_large_int(postgres_target):
     singer_file_to_target(file_name, postgres_target)
 
 
+def test_anyof(postgres_config_no_ssl, engine):
+    """Test that anyOf is handled correctly"""
+    table_name = "commits"
+    file_name = f"{table_name}.singer"
+    singer_file_to_target(file_name, TargetPostgres(config=postgres_config_no_ssl))
+    with engine.connect() as connection:
+        result: sqlalchemy.engine.cursor.LegacyCursorResult = connection.execute(
+            f"SELECT column_name, data_type FROM information_schema.columns WHERE table_name = '{table_name}'"
```

Comment:
This works, I think `https://github.com/MeltanoLabs/target-postgres/blob/main/target_postgres/connector.py#L57C17-L57C17`

Would be easier as you can use SQLAlchmey to do some of this for you, then you call meta[table_name].columns and then ask about each column individually

Reply:
Didn't use the exact same implementation with meta[table_name].columns, but it does now use SQLAlchemy to get datatypes.